src/main/scala/firrtl/Emitter.scala

```diff
@@ -205,6 +205,9 @@ class VerilogEmitter extends SeqTransform with Emitter {
   /** Turn Params into Verilog Strings */
   def stringify(param: Param): String = param match {
     case IntParam(name, value) => s".$name($value)"
+      val blen = value.bitLength
+      val lit = if (value > 0) s"$blen'd$value" else s"-${blen+1}'sd${value.abs}"
```

Comment:
Is this right when value is 0? (Or is that impossible?)

Reply:
Good catch, now it does work (at least using Verilator below), because this emits as follows for the test below:
```verilog
LargeParam #(.WIDTH(1'd1), .DATA(-1'sd0))
```
That being said, it's a little weird. Maybe should just handle zero explicitly to avoid confusion.